src/fontra/views/editor/panel-reference-font.js

```diff
@@ -322,6 +323,51 @@ export default class ReferenceFontPanel extends Panel {
         cssString;
       this.editorController.canvasController.requestUpdate();
     });
+
+    this.initActions();
+  }
+
+  initActions() {
+    const topic = "0200-action-topics.reference-font";
+    registerAction(
+      "action.select-previous-reference-font",
+      {
+        topic,
+        titleKey: "menubar.view.select-previous-reference-font",
+        defaultShortCuts: [],
+      },
+      () => this.doSelectPreviousNextReferenceFont(true)
+    );
+
+    registerAction(
+      "action.select-next-reference-font",
+      {
+        topic,
+        titleKey: "menubar.view.select-next-reference-font",
+        defaultShortCuts: [],
+      },
+      () => this.doSelectPreviousNextReferenceFont(false)
+    );
+  }
+
+  async doSelectPreviousNextReferenceFont(selectPrevious) {
+    const listLength = this.filesUIList.items.length;
+    if (!listLength) {
+      return;
+    }
+
+    const index = this.filesUIList.getSelectedItemIndex();
```

Comment:
```suggestion
    const index = this.filesUIList.getSelectedItemIndex() || 0;
```
Then we don't have to bother with isNaN below.